lib/ProductOpener/Display.pm

```diff
@@ -8654,6 +8663,14 @@ the rounded value according to the Nutri-Score rules, and the corresponding poin
 
 =cut
 
+my $config = {
+	INCLUDE_PATH => $data_root . '/templates',
+	INTERPOLATE => 1,
+	EVAL_PERL => 1,
+};
+
+my $tt = Template->new($config);
```

Comment:
This is already defined in https://github.com/openfoodfacts/openfoodfacts-server/blob/603b5f753816b79b4f7384d9b225359d2c523927/lib/ProductOpener/Display.pm#L242, so I think this won't work (or rather: should not be required).

Reply:
Oh yeah, forgot to remove this.